src/components/tx/ErrorMessage/index.tsx

```diff
@@ -46,7 +48,7 @@ const ErrorMessage = ({
 
           {error && showDetails && (
             <Typography variant="body2" className={css.details}>
-              {error.reason || error.message.slice(0, 300)}
+              {(error.reason || error.message).replace(ETHERS_PREFIX, '').trim().slice(0, 500)}
```

Comment:
Do you remember the reason why we sliced the message at 300 characters? Is it possibly a limitation of Sentry?

Reply:
Just to not blow it up visually.